Userland/Libraries/LibC/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-set(SOURCES
+# FIXME: These should be LibC sources required by DynamicLoader. While it surely needs just a
```

Comment:
This is interesting. I managed to reduce this to a smaller set here:
https://github.com/supercomputer7/serenity/commit/722a4ab4b677e1eb495b58685b33c23d17da2ff6
Is it not working because you build a library and not include the files directly in the DynamicLoader `CMakeLists.txt` file?

Reply:
Apparently I wasn't trying hard enough. I guess I descended 3 levels of transitive dependencies and gave up. Thanks for the list!